run.py

```diff
@@ -30,10 +47,18 @@ def main():
     lang = settings.value('Language')
     locale_folder = op.join(BASE_PATH, 'locale')
     install_gettext_trans_under_qt(locale_folder, lang)
+    # Handle OS signals
+    setUpSignals()
+    # Let the Python interpreter run every 500ms to handle signals.
+    from PyQt5.QtCore import QTimer
+    timer = QTimer()
```

Comment:
I don't understand this. What happens if we don't have this timer? Qt's runloop doesn't let signal handling work properly?